catalog/index.html

```diff
@@ -0,0 +1,344 @@
+
+      <section class="more-goods">
+        <div class="more-goods__wrapper container">
+          <table class="more-goods__table">
+            <tr class="more-goods__string">
+              <td class="more-goods__cell more-goods__name">Сахарозаменить</td>
+              <td class="more-goods__cell">1 Упаковка (100 г)</td>
+              <td class="more-goods__cell">200 р.</td>
+              <td class="more-goods__cell">
+                <button class="more-goods__btn button">Заказать</button>
+              </td>
+            </tr>
+            <tr class="more-goods__string">
+              <td class="more-goods__cell more-goods__name">Питьевая вода</td>
+              <td class="more-goods__cell">5 литров</td>
+              <td class="more-goods__cell">50 р.</td>
+              <td class="more-goods__cell">
+                <button class="more-goods__btn button">Заказать</button>
+              </td>
+            </tr>
+            <tr class="more-goods__string">
+              <td class="more-goods__cell more-goods__name">Молоко</td>
+              <td class="more-goods__cell">1 литр</td>
+              <td class="more-goods__cell">100 р.</td>
+              <td class="more-goods__cell">
+                <button class="more-goods__btn button">Заказать</button>
+              </td>
+            </tr>
+            <tr class="more-goods__string">
+              <td class="more-goods__cell more-goods__name">Витамины</td>
+              <td class="more-goods__cell">1 упаковка (30 г)</td>
+              <td class="more-goods__cell">300 р.</td>
+              <td class="more-goods__cell">
+                <button class="more-goods__btn button">Заказать</button>
+              </td>
+            </tr>
+          </table>
+          <div class="more-goods__gift">
+            <img
+              class="more-goods__gift-pic"
+              src="../img/gift.svg"
+              width="55"
+              height="65"
+              alt="Подарок"
+            />
+            <p class="more-goods__offer">
+              Закажите все и получите чехол для кота в подарок!
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="map">
+        <div class="map__content">
+          <p class="map__cooperation">Приглашаем к сотрудничеству дилеров!</p>
+          <div class="map__text-wrap">
+            <p class="map__adress">ул. Большая Конюшенная, д. 19/8</p>
+            <p class="map__adress">Санкт-Петербург</p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
+      <div class="footer__wrapper container">
+        <img
+          class="footer__logo"
+          src="../img/footer-logo.svg"
+          width="128"
+          height="24"
+          alt="Логотип кошачьей энергии"
+        />
+        <ul class="footer-socials">
+          <li class="footer__item">
+            <a class="footer__link" href="#"
+              ><img
+                class="footer__vk"
```

Comment:
Also don't think you should use different class names for different social icons — they seem to have similar styles.